api/v2/api.js

```diff
@@ -134,6 +134,8 @@ router.get('/random', async (req, res) => {
 
 });
 
+router.get('')
```

Comment:
What does this do?

Reply:
I think for this commit, I had renamed random to jumble and added the route for grumble, but didn't add anything to it yet